services/core/BACnetProxy/bacnet_proxy/agent.py

```diff
@@ -52,9 +52,7 @@
 bacnet_logger.setLevel(logging.WARNING)
 __version__ = '0.4'
 
-import os.path
-import errno
-from volttron.platform.agent import json as jsonapi
+from volttron.platform.agent.known_identities import PLATFORM_DRIVER
 from collections import defaultdict
```

Comment:
imports from volttron should be below the native installed and third party installed imports